static/js/profile.js

```diff
@@ -34,24 +34,25 @@ var vm = new Vue({
 
 function update_data(runs, user) {
             
-    vm.totalratedruns = runs.length;
-    vm.email = user[0]['email'];
-    if (user[0]['email_confirmed']) {
+    vm.totalratedruns = runs['total_prompts'];
+    vm.email = user['email'];
```

Comment:
What happens when the user[`email`] isn't in the response? I think this would error, but I'd rather just not show the email field in the first place